RedHat-UX/red-hat-design-system
# docs: playground loader

Closes #581
 
## What I did

1. removed the rhds bundle from the base template
    - if you want to load an element now, add it to the `importElements` list in frontmatter
    - all pages load cta and footer
    - element pages also load subnav, and their own element by tagname
1. fixed the cta shortcode when it contains markdown
1. Added a (docs-only) `<rh-playground>` custom element
    - it's got a button!
    - it's got a spinner!
    - it's got some animations and css filters!
    - it simplifies and encapsulates the demo loading code! 


## Testing Instructions
We're looking for undefined elements and missing icons
You can get a list of undefined elements like so:
```js
$$(':not(:defined)')
```
to see all the pf-icons on the page, use the find function in element inspector (ctrl-f) and type `pf-icon`

1. check the main page
2. check 'static' pages like `/foundations/`
3. check all the element pages
